CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 FEATURES:
 
 * **New Resource:** sumologic_hierarchy (GH-260)
+* **New Resource:** cse_newtwork_block
```

Comment:
spelling is off here

Reply:
thank you @josh-williams ; done under https://github.com/SumoLogic/terraform-provider-sumologic/pull/271/commits/cac0ca4be3488a17ddb6b7c3826bff12d2c43aa6